src/Microsoft.IdentityModel.Tokens/Validation/Validators.IssuerSigningKey.cs

```diff
@@ -17,17 +17,15 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     /// <param name="signingKey">The security key to validate.</param>
     /// <param name="securityToken">The <see cref="SecurityToken"/> that is being validated.</param>
-    /// <param name="validationParameters">The <see cref="TokenValidationParameters"/> to be used for validating the token.</param>
-    /// <param name="callContext"></param>
-    /// <param name="cancellationToken"></param>
```

Comment:
Is there a reason to remove cancellation token now?

Reply:
It is not an async call.

Reply:
Please reply instead of editing the original comment as it makes the conversation difficult to follow.
If/when we make the call async we will need to change the return type to `Task` and likely use a separate method. On that one we can add a cancellation token.